sdcm/tester.py

```diff
@@ -162,7 +162,7 @@ def setUp(self):
         self.db_cluster.wait_for_init()
         db_node_address = self.db_cluster.nodes[0].private_ip_address
         self.loaders.wait_for_init(db_node_address=db_node_address)
-        nodes_monitored = [node.public_ip_address for node in self.db_cluster.nodes]
+        nodes_monitored = [node.private_ip_address for node in self.db_cluster.nodes]
```

Comment:
The nodes_monitored is the source of ips that will be used in monitor related stuffs, so we only need to change this point.